### DISCUSSION TITLE
carla install - make launch

### DISCUSSION THREAD

**Discussion by SJ824** [carla-simulator/carla#6759] [Q&A]
I've done all the installs, run make launch, but I keep getting this error 'fatal error: 'Renderer/Public/GBufferView.h' file not found'. 

The previous 'libpng-1.6.37.tar.xz' error was fixed with 
- wget https://download.sourceforge.net/libpng/libpng-1.6.37.tar.xz
- tar xvf libpng-1.6.37.tar.xz

I solved it like this: 'fatal error: 'Renderer/Public/GBufferView.h' file not found' I don't know how to solve this error. 

'4 warnings and 1 error generated.
4 warnings and 1 error generated.
5 warnings and 1 error generated.
4 warnings and 1 error generated.
4 warnings and 1 error generated.
make[1]: *** [Makefile:315: CarlaUE4Editor] Error 6
make[1]: Leaving directory '/home/carla/Unreal/CarlaUE4'
make: *** [Util/BuildTools/Linux.mk:7: launch] Error 2

How do I fix this?

**Comment by chirudeep001** [comment]
check this out:
https://youtu.be/lLkFA0fPrgs?si=bZio1jEElBGn6wHd
